voice.js

```diff
@@ -0,0 +1,183 @@
+let personName = "";
+let toDoList = [];
+
+function greetWithName(commandArray) {
```

Comment:
```suggestion
function greetWithName(commandWords) {
```
As mentioned elsewhere, this variable name is confusing.